src/server/scripts/Northrend/ChamberOfAspects/RubySanctum/boss_halion.cpp

```diff
@@ -1006,10 +1024,7 @@ class npc_meteor_strike_initial : public CreatureScript
                         me->SetOrientation(angle[i]);
                         Position newPos = me->GetNearPosition(10.0f, 0.0f); // Exact distance
                         if (Creature* meteor = me->SummonCreature(NPC_METEOR_STRIKE_NORTH + i, newPos, TEMPSUMMON_TIMED_DESPAWN, 30000))
-                        {
-                            meteor->SetOrientation(angle[i]);
```

Comment:
I never tested those changes i did with the random adjusting because my pc is way too shitty but I clearly remember that this line was needed for the fire spread to go in the correct direction and not randomly fuck over the terrain. Even though that's what it should be, orientation is used to determine the randomness in the position of the next spawn, so i'm not sure why this is gone?